docs/app-dev/abci-cli.md

```diff
@@ -213,9 +214,137 @@ we do `deliver_tx "abc=efg"` it will store `(abc, efg)`.
 Similarly, you could put the commands in a file and run
 `abci-cli --verbose batch < myfile`.
 
+## Counter - Another Example
```

Comment:
I think we have removed Counter App in `main`, and since we have merge `main` into `feature/abci++ppp` it also applies to the latter